cli/main.go

```diff
@@ -112,13 +115,25 @@ func Main() {
 		log.Fatal("Please specify a relay timeout greater than 0")
 	}
 
+
+	if *relayMinBidEth < 0.0 {
+		log.Fatal("Please specify a non-negative minimum bid")
+	}
+
+	if *relayMinBidEth > 100.0 {
```

Comment:
I don't see any reason to hardcode an arbitrary max minBid. I think it should be up to the user to define any valid positive number. Additionally mev boost could be run on future forks where a high absolute number could be normal.

Reply:
The motivation was to catch user mistakes in the flag (e.g. entering the value in wei instead of eth).  I have no problem removing it or setting it much higher.

Reply:
Raised to 1,000,000 Ethers